packages/components/app/styles/components/dropdown.scss

```diff
@@ -25,311 +25,385 @@ $hds-dropdown-toggle-base-height: 36px;
-    color: var(--token-color-foreground-strong);
+    border-radius: 5px;
+    border: 1px solid var(--token-color-border-primary);
+    box-sizing: border-box;
+    color: var(--token-color-foreground-primary);  
     display: flex;
-    height: $hds-dropdown-toggle-base-height;
-    isolation: isolate;
-    justify-content: center;
-    outline-style: solid; // used to avoid double outline+focus-ring in Safari (see https://github.com/hashicorp/design-system-components/issues/161#issuecomment-1031548656)
-    outline-color: transparent; // We need this to be transparent for a11y
-    padding: 0;
-    position: relative;
-    min-width: $hds-dropdown-toggle-base-height;
-
+    font-family: var(--token-typography-font-stack-code);
+    justify-content: space-between;
+    padding: 12px 8px;
+    width: 100%;
+    
     &:hover,
     &.is-hover {
-      background-color: var(--token-color-surface-interactive);
-      border-color: var(--token-color-border-strong);
+      background-color: var(--token-color-surface-interactive-hover);
       cursor: pointer;
     }
 
     &:focus,
     &.is-focus {
-      @include hds-focus-ring-with-pseudo-element(
-        $top: -1px,
-        $right: -1px,
-        $bottom: -1px,
-        $left: -1px,
-        $radius: $hds-dropdown-toggle-border-radius
-      );
+      //TODO this focus is just way too complex 
+      background-color:  var(--token-color-surface-action);
+      border-color: var(--token-color-focus-action-internal);
     }
-
-    &:active,
+    
+    &:active, 
     &.is-active {
       background-color: var(--token-color-surface-interactive-active);
-      border-color: var(--token-color-border-strong);
     }
-  }
 
-  .hds-dropdown-toggle-icon__wrapper {
-    align-items: center;
-    border-radius: 3px; // $hds-dropdown-toggle-border-radius- 1px padding - 1px border
-    display: flex;
-    justify-content: center;
-    height: 32px;
-    padding: 1px;
-    width: 32px;
-
-    .hds-button__icon {
-      margin-right: -6px; // we apply a negative margin to counter the padding-right of the button and reduce the visual space between the icon and the right border
-      margin-left: 0.5rem; // this overrides the rule `.hds-button__text + .hds-button__icon`
+    &.is-success {
+      border-color: var(--token-color-border-success);
+      background-color: var(--token-color-surface-success);
+      .hds-dropdown-list-item__copy-item-icon {
+        color: var(--token-color-foreground-success);
+      }
+    } 
+
+    .hds-dropdown-list-item__copy-item-text {
+      font-size: var(--token-typography-code-100-font-size);
+      font-weight: var(--token-typography-font-weight-regular);
+      line-height: var(--token-typography-code-100-line-height);
+      max-width: 250px; // TODO we should be able to figure out the proportions here
+      overflow: hidden;
+      text-align: left;
+      text-overflow: ellipsis;
```

Comment:
[issue/blocker]: here is missing `white-space: nowrap;` to work correctly (otherwise it will be multi-line)